build/configs/imxrt1020-evk/nxp_demo/defconfig

```diff
@@ -345,7 +345,7 @@ CONFIG_PREALLOC_TIMERS=8
 # Tasks and Scheduling
 #
 CONFIG_INIT_ENTRYPOINT=y
-CONFIG_RR_INTERVAL=0
+CONFIG_RR_INTERVAL=100
```

Comment:
You are enabling this to avoid failure with timer example, how about creating a dependency with CONFIG_EXAMPLES_TIMER??

Reply:
@sijupunnnoose @yashwanth686007 Thank you for explaining the reason of this. I also thought this suggestion. But I want to know is it really needed? Without RR, isn't it working?

Reply:
@sijupunnnoose @sunghan-chang , I believe its not only with timer example, its a generic round robin functionality that doesn't take place without this interval.
By the current default zero value, round robin scheduling will be disabled. If RR interval is set to zero, the task stays on the head of the ready-to-run list until it completes. It doesn't give any chance to other tasks of equal priority to run.

Reply:
@sunghan-chang, in master timer example fails with message "failed to set policy"

ret = pthread_attr_setschedpolicy(&attr, SCHED_RR);
if (ret != 0) {
	fprintf(stderr, "failed to set policy(%d)\n", ret);
	return -ret;
}

Reply:
@yashwanth686007 We support FIFO as well as RR. Someone could want to work with FIFO. To work those defconfig, is RR necessary? I don't think so.
@sijupunnnoose  Current timer example makes it failed with FIFO. But it can't be the reason. I think we can use FIFO to run. Timer should be working with FIFO also. I think our timer example was implemented wrongly. We don't need to force to turn it on.

Reply:
@sunghan-chang , yes, user can either of the scheduling policies. For this defconfig, RR is not necessary. Since we followed RR in remaining defconfigs, I have made the changes. If not required, i will revert.

Reply:
@yashwanth686007 I think this is unnecessary. If you have the another reason, please let me know.

Reply:
Okay.. I'll close the PR. Thanks.